src/Game.py

```diff
@@ -7,6 +7,11 @@ def __init__(self):
         self.parser = ParseInput()
 
     def run(self) -> int:
-        self.parser.askInput()
+        try:
```

Comment:
Printing the error is not enough to handle it. We need to see with an APE or with Hugo how to handle these exceptions